docs/csharp/language-reference/operators/patterns.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Patterns - C# reference"
+description: "Learn about the patterns supported by C# switch and is expressions and C# switch statement - C# reference"
+ms.date: 02/15/2021
+helpviewer_keywords: 
+  - "pattern matching [C#]"
+---
+# Patterns (C# reference)
+
+Intro.
+
+## Declaration and type patterns
+
+The *declaration pattern* has the following form:
+
+```csharp
+T v
+```
```

Comment:
Throughout this article, I'd like to see examples rather than the spec style of `T`, `C`, `v`, and so on.

We've used the canonical `Person`, `Student`, `Teacher` for a reasonably simple type. Other ideas are welcome.  

I think that concrete examples will make explaining the concepts throughout the article easier. My hope is that you'll find less need to repeat the normative language in the spec.

Reply:
@BillWagner I agree: using `T`-`C`-`v` language is less preferable than examples and I also think examples are clearer. At the same time, I think in this particular case (declaration pattern) that spec-style notation is fine. But I don't want to use it for another patterns. 

In my updated version, I've tried to reduce such a language anyway. Please review it and let me know what you think.

Reply:
I like it. I'm still noticing the few places where it does exist. I'm fine merging this now, and giving that one final look when the full package is ready.